src/secp256k1.c

```diff
@@ -91,6 +99,7 @@ secp256k1_context* secp256k1_context_clone(const secp256k1_context* ctx) {
 }
 
 void secp256k1_context_destroy(secp256k1_context* ctx) {
+    CHECK(ctx != secp256k1_context_no_precomp);
```

Comment:
Are these necessary? A non-`const` ctx pointer is used, which can't be `secp256k1_context_no_precomp` (as that one is const).